dev-resources/template/2_lrs.yml

```diff
@@ -0,0 +1,536 @@
+          Action: ['sts:AssumeRole']
+      Path: /
+      ManagedPolicyArns:
+        - !Ref InstancePolicy
+
+  InstanceProfile:
+    Type: AWS::IAM::InstanceProfile
+    Properties:
+      Path: /
+      Roles: [!Ref InstanceRole]
+
+  InstanceSG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Access to the bastion
+      VpcId: !Ref VPCId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: '73.133.206.211/32'
+        - IpProtocol: tcp
+          FromPort: 8080
+          ToPort: 8080
+          CidrIp: '73.133.206.211/32'
+
+
+  DBInstanceIngress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from the Bastion instance to RDS instance
+      GroupId: !Ref DBInstanceSG
+      IpProtocol: tcp
+      FromPort: !Ref DBPort
+      ToPort: !Ref DBPort
+      SourceSecurityGroupId: !Ref InstanceSG
+
+  LrsInstance:
+    Type: AWS::EC2::Instance
+    DependsOn: InitDBCustomResource
+    CreationPolicy:
+      ResourceSignal:
+        Timeout: PT15M
+    Metadata:
+      AWS::CloudFormation::Init:
+        configSets:
+          default:
+            - 00_installLrs
+            - 01_setupCfnHup
+            - 02_configureLrs
+            - 03_enableLrs
+            - 04_startLrs
+          UpdateEnvironment:
+            - 00_installLrs
+            - 02_configureLrs
+            - 05_restartLrs
+        00_installLrs:
+          files:
+            "/tmp/lrsql.zip":
+              source: !Join
+                - '/'
+                - - "https://github.com/yetanalytics/lrsql/releases/download"
+                  - !Ref LrsVersion
+                  - "lrsql.zip"
+              mode: "000755"
+              owner: "root"
+              group: "root"
+          commands:
+            00make_dir:
+              cwd: "~"
+              command: "sudo mkdir /opt/lrsql"
+            01unzip_lrs:
+              cwd: "~"
+              command: "sudo unzip /tmp/lrsql.zip -d /opt/lrsql/"
+        01_setupCfnHup:
+          files:
+            '/etc/cfn/cfn-hup.conf':
+              content: !Sub |
+                [main]
+                stack=${AWS::StackId}
+                region=${AWS::Region}
+                interval=1
+              mode: '000400'
+              owner: root
+              group: root
+            "/lib/systemd/system/cfn-hup.service":
+              content: !Sub |
+                [Unit]
+                Description=cfn-hup daemon
+                [Service]
+                Type=simple
+                ExecStart=/opt/aws/bin/cfn-hup
+                Restart=always
+                [Install]
+                WantedBy=multi-user.target
+            '/etc/cfn/hooks.d/tc-auto-reloader.conf':
+              content: !Sub |
+                [cfn-auto-reloader-hook]
+                triggers=post.update
+                path=Resources.BastionInstance.Metadata.AWS::CloudFormation::Init.02_configureLrs
```

Comment:
This is LrsInstance but this auto-reloader is trying to use `BastionInstance` from TC. It won't work so good!

Reply:
done - thought i got all of those!